app/controlplane/internal/server/grpc.go

```diff
@@ -212,7 +212,7 @@ func craftMiddleware(opts *Opts) []middleware.Middleware {
 			// 2.a - Set its workflow and organization in the context
 			usercontext.WithAttestationContextFromRobotAccount(opts.RobotAccountUseCase, opts.OrganizationUseCase, logHelper),
 			// 2.b - Set its API token and Robot Account as alternative to the user
-			usercontext.WithAttestationContextFromAPIToken(opts.APITokenUseCase, logHelper),
+			usercontext.WithAttestationContextFromAPIToken(opts.APITokenUseCase, opts.OrganizationUseCase, logHelper),
```

Comment:
this is something that we were missing in our middleware. Technically it's not from this patch. I needed it in previous implementations of it. If you want I could try to move it outside.

Reply:
I think it's fine here, no worries, thanks!!